wp-content/themes/chuongDemo/header.php

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
take a look on Template File Checklist, https://codex.wordpress.org/Theme_Development, to create header.php